### ISSUE TITLE
css.properties.visibility - Collapse working correctly in FF/Chrome/Opera for windows

### ISSUE THREAD

**Issue by DavePlater** [mdn/browser-compat-data#10038]
#### What information was incorrect, unhelpful, or incomplete?
Implementation notes for firefox, chrome and opera on PC say that "collapse" is treated like "hidden" and leaves white space.

#### What did you expect to see?
Implementation notes for safari on iOS also says that "collapse" is treated like "hidden" and leaves white space.

#### Did you test this? If so, how?
Testing "collapse" on TBODY elements in Firefox88, Chrome89 and Opera75 on windows10, did not observe white space, was not being treated like "hidden"
Testing "collapse" on TBODY elements in safari on iOS had a white space and was similar to "hidden"

<details>
<summary>MDN page report details</summary>

* Query: `css.properties.visibility`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/visibility
* Report started: 2021-04-22T21:07:20.701Z

</details>

**Comment by queengooborg** [comment]
Sorry, it's hard to follow what changes are needed to be made as the issue template was not filled out properly.  If this is still an issue, please feel free to submit a new issue with the updated template and fill out each section of the issue template according to the requested info.
